src/store/__mocks__/index.ts

```diff
@@ -12,8 +12,10 @@ module.exports = {
     getCreatorStagedForLocation: jest.fn(),
     getCreatorStatus: jest.fn(),
     getFileId: jest.fn().mockReturnValue('0'),
```

Comment:
I'm a little dubious of having these default values in a global store mock. It seems like we should be able to mock the entire store and then set the return values in each test.

Reply:
Maybe we can change all of the store mocks in another pr.

Reply:
That's fine. Can we avoid adding to the problem in this PR?

Reply:
Sure